docs/streams/developer-guide/dsl-api.html

```diff
@@ -901,25 +929,28 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                             <col width="78%" />
                         </colgroup>
                         <thead valign="bottom">
-                        <tr class="row-odd"><th class="head">Transformation</th>
+                        <tr class="row-even"><th class="head">Transformation</th>
                             <th class="head">Description</th>
                         </tr>
                         </thead>
                         <tbody valign="top">
-                        <tr class="row-even"><td><p class="first"><strong>Aggregate</strong></p>
+                        <tr class="row-odd"><td><p class="first"><strong>Aggregate</strong></p>
                             <ul class="last simple">
                                 <li>KGroupedStream &rarr; KTable</li>
                                 <li>KGroupedTable &rarr; KTable</li>
                             </ul>
                         </td>
-                            <td><p class="first"><strong>Rolling aggregation.</strong> Aggregates the values of (non-windowed) records by the grouped key.
+                            <td><p class="first"><strong>Rolling aggregation.</strong> Aggregates the values of (non-windowed) records by the grouped key or cogrouped.
                                 Aggregating is a generalization of <code class="docutils literal"><span class="pre">reduce</span></code> and allows, for example, the aggregate value to have a different
                                 type than the input values.
                                 (<a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/KGroupedStream.html">KGroupedStream details</a>,
                                 <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/KGroupedTable.html">KGroupedTable details</a>)</p>
```

Comment:
The `</p>` tag at the end seem wrong here, as the paragraph is (correctly) closed in the next line.

Reply:
you are right